pkg/operator/controller/ingress/controller.go

```diff
@@ -453,6 +453,11 @@ func setDefaultPublishingStrategy(ic *operatorv1.IngressController, platformStat
 			effectiveStrategy.LoadBalancer.DNSManagementPolicy = operatorv1.UnmanagedLoadBalancerDNS
 		}
 
+		// OpenStack platform does not support managed DNS for load balancers.
+		if platformStatus.Type == configv1.OpenStackPlatformType && platformStatus.OpenStack != nil {
```

Comment:
I think this should be unit tested in `TestSetDefaultPublishingStrategySetsPlatformDefaults`. I looked at the test, be careful with `makePlatformStatus`, it doesn't set `platformStatus.OpenStack` so this logic doesn't get exercised.

Reply:
oh thanks, I missed that.